plugins/helphub-post-types/classes/class-helphub-post-types-post-type.php

```diff
@@ -225,26 +237,25 @@ public function register_custom_column_headings( $defaults ) {
 	 * @return array           Modified array.
 	 */
 	public function updated_messages( $messages ) {
-		global $post;
+		global $post, $post_ID; /* @codingStandardsIgnoreLine */
```

Comment:
`$post_ID` should be ignored as part of `WordPress.Variables.GlobalVariablesSniff` though for some reason it isn't (See also `WordPress.NamingConventions.ValidVariableName.NotSnakeCase`)

For now lets keep `/* @codingStandardsIgnoreLine */` here, we can revisit and audit all the PHPCS ignores in a follow up PR.